ext/date/tests/gmstrftime_variation20.phpt

```diff
@@ -35,18 +35,18 @@ foreach($inputs as $key =>$value) {
 
 --Newline character--
 
-Deprecated: Function gmstrftime() is deprecated in %s on line %d
+Deprecated: Function gmstrftime() is deprecated since 8.1, use IntlDateFormatter::format() instead in %s on line %d
 string(1) "
 "
 
-Deprecated: Function gmstrftime() is deprecated in %s on line %d
+Deprecated: Function gmstrftime() is deprecated since 8.1, use IntlDateFormatter::format() instead in %s on line %d
 string(1) "
 "
 
 --Tab character--
 
-Deprecated: Function gmstrftime() is deprecated in %s on line %d
-string(1) "%r\s%r"
+Deprecated: Function gmstrftime() is deprecated since 8.1, use IntlDateFormatter::format() instead in %s on line %d
+string(1) "	"
 
-Deprecated: Function gmstrftime() is deprecated in %s on line %d
-string(1) "%r\s%r"
+Deprecated: Function gmstrftime() is deprecated since 8.1, use IntlDateFormatter::format() instead in %s on line %d
+string(1) "	"
```

Comment:
I'm guessing the test change makes sense, or can this be different depending on platform?

Reply:
Good catch, will check. This is an unintentional change created by `--bless` (I already had to revert one of these).

Reply:
The test change looks correct to me. I assume this was written like this to prevent issues with editors that are configured to replace tabs with spaces.

Nevertheless I reverted the change: It is unrelated to the topic of this PR and it makes sense to me to minimize churn for functionality that is deprecated anyways.